tools/distpartitioning/data_shuffle.py

```diff
@@ -337,12 +357,18 @@ def exchange_graph_data(rank, world_size, node_features, node_feat_tids, edge_da
         in the world. The edge data is received by each rank in the process of data shuffling.
     """
     rcvd_node_features, rcvd_global_nids = exchange_node_features(rank, world_size, node_feat_tids, \
-                                                ntypes_gnid_range_map, id_lookup, node_features)
+                                                ntypes_gnid_range_map, id_lookup, node_features, \
+                                                constants.STR_NODE_FEATURES, None)
     logging.info(f'[Rank: {rank}] Done with node features exchange.')
 
+    rcvd_edge_features, rcvd_global_eids = exchange_node_features(rank, world_size, edge_feat_tids, \
```

Comment:
pls rename the called function, this is for edge feature exchange?

Reply:
taken care of.